Source/RxSwift/Observable+Moya.swift

```diff
@@ -1,5 +1,8 @@
 import Foundation
 import RxSwift
+#if !COCOAPODS
```

Comment:
I talked to orta about it https://github.com/CocoaPods/CocoaPods/issues/5889#issuecomment-247737624, he said that Cocoapods defines a env variable, that I think (but am not sure) can be used in code with a `#if` check.

We could also define our own flag, but rn that just seems like extra work to maintain for no added cost. If theres another reason to define the flag, then we should jump on it 😄